dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM circleci/node:10.14.1 as builder
+
+WORKDIR /usr/src/app
+
+COPY /src /usr/src/app/src
+COPY /package.json /usr/src/app/package.json
+COPY /tsconfig.json /usr/src/app/tsconfig.json
+COPY /yarn.lock /usr/src/app/yarn.lock
+
+RUN sudo chmod -R 777 /usr/src/app \
+  && yarn install \
+  && yarn build
+
+FROM node:10.14.1-alpine
```

Comment:
Why do I see two FROM? What's this needed for?

Reply:
As I said, I followed the same approach used in the [app IO backend](https://github.com/teamdigitale/io-backend). Here two containers are used: the first one is used to build the app (transpile the typescript files) and the second is used to run the app after copying the transpiled files from the first one.